docusaurus.config.js

```diff
@@ -170,7 +170,7 @@ const config = {
       prism: {
         theme: lightTheme,
         darkTheme: darkTheme,
-        additionalLanguages: ["java", "gradle", "toml", "groovy", "kotlin"],
+        additionalLanguages: ["java", "gradle", "toml", "groovy", "kotlin", "javascript", "json", "json5"],
```

Comment:
This is for the syntax highlighting correct? If so, You can resolve this comment.